scripts/ci-install-macos-latest.sh

```diff
@@ -1,6 +1,7 @@
 brew update
 brew install llvm --HEAD
 brew install neovim --HEAD
+brew install tree-sitter
```

Comment:
This seems to be included already in `brew install neovim --HEAD`

Reply:
Have removed it locally

Reply:
https://github.com/theHamsta/nvim-treesitter/blob/62bda244c9bf7e88f31d119f72bceae0e7116443/lua/nvim-treesitter/install.lua#L157

It does not work due to this line. Sorry for putting it there in the first place. I guess the logic should be if `not requires_generate_from_source or (not vim.fn.has('win32') and not generate_requires_npm)`

Reply:
@nifoc maybe I was wrong...   maybe just `requires_generate_from_grammar` -> `generate_requires_npm` as the message says.

Reply:
I hate CIs... :sweat_smile:

Reply:
hmmm … If it change the line to `repo.generate_requires_npm` I think it would not install `typescript` (+ others)?
I only checked for `typescript` and it's currently installed in CI.

Reply:
You're right... Can this check be removed completely?

Reply:
@nifoc this seems to have worked. All parsers are now either generatable without npm or installable directly from C files.